### ISSUE TITLE
v20150519 stalls when upgrading with Homebrew

### ISSUE THREAD

**Issue by ddarbyson** [sstephenson/ruby-build#763]
When updating with Homebrew `$ brew update` version 20150506 to 20150519 seems to stall during install:

```
==> Upgrading ruby-build
==> Downloading https://github.com/sstephenson/ruby-build/archive/v20150519.tar.gz
######################################################################## 100.0%
==> ./install.sh
```

it will hang here...

I've tested with Yosemite 10.10.3 on two separate machines. Both are stalling during install.

**Comment by mislav** [comment]
Can't reproduce. Also on 10.10.3. There isn't anything in the new release that should make it stall. Does it happen every time?

**Comment by ddarbyson** [comment]
Yeah, I tested on 4 machines. I have rbenv installed with ruby 2.0.0p643. Not sure if anything is conflicting there, but I thought I’d mention it.

**Comment by mislav** [comment]
[These are literally the only changes](https://github.com/sstephenson/ruby-build/compare/v20150506...v20150519) between versions. Hardly anything that can lock up your computer.

[ruby-build uses `install` command](https://github.com/sstephenson/ruby-build/blob/a5d0479af7ca59e9b31f52156d6b38887f68bf24/install.sh#L19-20) to install its files into Homebrew's Cellar. You could look into whether that command works correctly in recursive mode.

**Comment by mislav** [comment]
I didn't receive reports from other people that they can't upgrade. Is this still affecting you?

**Comment by ddarbyson** [comment]
I looked into `install.sha and you were right, there is no suspected cause for any commands to be stalling out. However, I did notice when opening up Activity Monitor that there were many Ruby process running. Plus 30 at least.

Anyway, I ended up cloning into the plugins directory and skipped Homebrew all together to work around this issue. 

If I had more time I would look into this for you, however no one is reporting this issue so it must be tied to my machine only. Strangely enough, everything was working prior to ruby-build latest update. I would say close this issue out.
